src/patternfly/components/JumpLinks/jump-links.scss

```diff
@@ -152,12 +152,15 @@
 
 // stylelint-disable
 .pf-c-jump-links__item {
-  &.pf-m-current {
+  &.pf-m-current > .pf-c-jump-links__link {
     --pf-c-jump-links__link--before--BorderTopWidth: var(--pf-c-jump-links__item--m-current__link--before--BorderTopWidth);
     --pf-c-jump-links__link--before--BorderLeftWidth: var(--pf-c-jump-links__item--m-current__link--before--BorderLeftWidth);
     --pf-c-jump-links__link--before--BorderColor: var(--pf-c-jump-links__item--m-current__link--before--BorderColor);
     --pf-c-jump-links__link-text--Color: var(--pf-c-jump-links__item--m-current__link-text--Color);
   }
+  &.pf-m-current > .pf-c-jump-links__list {
+    margin-top: 0;
```

Comment:
This will cause the nested list to shift as the top-level item is made current.

![Nov-30-2020 10-48-46](https://user-images.githubusercontent.com/35148959/100638440-a87de300-32f9-11eb-8289-99ef5f350d78.gif)

The left grey border on a nested list doesn't need to be there. If you can disable it somehow, that will fix the border overlap.

Reply:
Fixed, thanks!